consensus/hotstuff/votecollector/combined_vote_processor_v2.go

```diff
@@ -30,15 +30,15 @@ import (
 // by `votecollector.VoteProcessorFactory` which adds the logic to verify
 // the proposer's vote (decorator pattern).
 type combinedVoteProcessorFactoryBaseV2 struct {
-	committee   hotstuff.Committee
+	committee   hotstuff.Replicas
```

Comment:
Just wondering, why do we only want to accept votes from un-ejected replicas? We will accept QCs with votes from ejected replicas, so these QCs are considered valid. 

I understand that we _can_ reject votes from ejected replicas here, so I'm guessing the argument is to err on the side of caution because it's just as easy to use the dynamic committee in this case. But is there any particular attack we're avoiding by doing this?